src/Components/App/App.js

```diff
@@ -69,6 +70,30 @@ class App extends Component {
             );
           }}
         />
+feature/login-page
+        <Switch>
```

Comment:
I am not sure I understand Switch completely, would you help me understand why it is needed here and why we need two Routes for the SingleMovie component?